scripting/get5/matchconfig.sp

```diff
@@ -882,6 +882,7 @@ public Action Command_AddCoach(int client, int args) {
     }
 
     if (!ConvertAuthToSteam64(auth, steam64)) {
+      ReplyToCommand(client, "Failed to parse steam ID: %s", auth);
```

Comment:
This is already called in a LogError, do we want to tell users if we fail to parse steam IDs? If so, I would recommend doing `ConvertAuthToSteam64(auth, steam64, false)` to avoid double reports in the server console.

Reply:
User cannot see error logs. It also would not make sense to continue with the rest of the function of the auth is invalid.

I think doing both is fine. Perhaps changing the Log to a LogDebug inside the auth converter?

Reply:
Ah, it shows error log in console?

We can remove the Reply perhaps, but it needs to return if false.

Reply:
I also was a bit quick here. I have now modified it so that a false return either `AddPlayerToTeam` or `AddCoachToTeam` prints out a more accurate response.

I also removed the check you commented on, since it's already being done inside the `AddCoachToTeam` itself. We don't need it outside of it.